source/androidx.media3/media3-common/Transforms/Metadata.xml

```diff
@@ -18,17 +18,24 @@
         Java.Lang.Object
     </attr>
 
-
     <attr
         path="/api/package[@name='androidx.media3.common']/interface[@name='Player.Listener']/method[@name='onCues' and count(parameter)=1 and parameter[1][@type='java.util.List&lt;androidx.media3.common.text.Cue&gt;']]"
         name="managedName"
         >
         OnCuesDeprecated
-    </attr>
-    <attr
-        path="/api/package[@name='androidx.media3.common']/interface[@name='Player.Listener']/method[@name='onPositionDiscontinuity' and count(parameter)=1 and parameter[1][@type='int']]"
-        name="managedName"
-        >
-        OnPositionDiscontinuityDeprecated
-    </attr>
+	  </attr>
+
+    <!-- Remove these as they cause issues with Media3.ExoPlayer-->
+    <remove-node
+        path="/api/package[@name='androidx.media3.common']/interface[@name='Player.Listener']/method[@name='onPositionDiscontinuity' and count(parameter)=3 and parameter[1][@type='androidx.media3.common.Player.PositionInfo'] and parameter[2][@type='androidx.media3.common.Player.PositionInfo'] and parameter[3][@type='int']]"
+        />
+
+  <!-- Abstract Player.Listener so that interface members have to be implemented.-->
+   <attr 
```

Comment:
I doubt this metadata does anything, as it sets the `abstract` attribute to `PlayerListenerAbstract`, which is not a valid `true`/`false` value.

Reply:
I am testing removing it now.